tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -25,12 +25,12 @@ class Homescreen(Base):
     _divider_locator = (By.CSS_SELECTOR, 'section.divider')
     _divider_separator_locator = (By.CSS_SELECTOR, 'section.divider .separator > span')
     _exit_edit_mode_locator = (By.ID, 'exit-edit-mode')
+    _homescreen_all_icons_locator = (By.CSS_SELECTOR, 'gaia-grid .icon:not(.placeholder)')
 
     _body_dragging_locator = (By.CSS_SELECTOR, 'body.dragging')
     _apps_locator = (By.ID, 'apps')
     _app_icon_locator = (By.CSS_SELECTOR, 'gaia-app-icon[data-identifier="%s"]')
-    _remove_locator = (By.ID, 'remove')
-
+    
```

Comment:
Pylint is a good command line tool. https://bugzilla.mozilla.org/show_bug.cgi?id=1186388 tracks how we could make it run on every commit.